codelabs/template.md

```diff
@@ -0,0 +1,56 @@
+author: YOUR NAME
+summary: CODELAB SUMMARY
+id: TEMPLATE-CODELAB-ID
+categories: codelab,blockly,MORE CATEGORIES
+status: Draft
+Feedback Link: https://github.com/google/blockly-samples/issues/new
+
+# CODELAB TITLE
+
+## Codelab overview
+
+### What you'll learn
+Describe the skills or tools the developers will learn by the end of this codelab.
+
+### What you'll build
+Describe what the finished product will look like.  Include screenshots or gifs.
+
+### Prerequisites
```

Comment:
From the other codelabs i've look at they all have "What you'll need" Not sure if we want to stick with that convention or not.

Reply:
Done